### DISCUSSION TITLE
volar inlay hints

### DISCUSSION THREAD

**Discussion by Kabuda-czh** [vuejs/language-tools#2066] [Q&A]
when i open vscode's typescript > inlay hints and use volar.
![1](https://user-images.githubusercontent.com/62021214/198211572-e9bef551-3085-45e2-8f12-17c090d36d96.png)

After some times, there will be repeated.
![2](https://user-images.githubusercontent.com/62021214/198211657-78627dc1-3571-4d1e-861a-873cdc2d3d4a.png)

if `restart Vue Server` it will fixed , is this volar's bug?

**Comment by johnsoncodehk** [comment]
Please track #1807
